build.sbt

```diff
@@ -551,6 +563,7 @@ lazy val sdk = (project in file("modules/sdk"))
     scalafixCommonSettings,
     commonSettings,
     commonTestSettings,
+    publish / skip := false,
```

Comment:
do I understand it correctly that only the sdk module would be published? if so, that's correct :+1:

Reply:
yes, only sdk will publish (this was confirmed on the Sonatype end as well). It is a bit of an odd syntax but this line is overriding the `publish / skip := true` value set in `commonSettings` (which is used by nearly every module). If we'd prefer to update all the modules definitions instead, I can do that.

Reply:
No, it's fine for me. Just wanted to make sure